README.md

```diff
@@ -0,0 +1,53 @@
+# Flatpak for DOSBox
+
+## Installation
+
+This Flatpak is available on
+[Flathub](https://flathub.org/apps/details/com.dosbox.DOSBox).
+After following the [Flatpak setup guide](https://flatpak.org/setup/),
```

Comment:
Why after? Following the setup guide should be sufficient. There is no need to run additional commands and no need to install this Flatpak under `--user`.

Reply:
I changed it to remove the `--user` switch. I left the rest of the paragraph for the same reason outlined below.

Reply:
Oops, I misread the "Flatpak setup guide" part and thought that you mean setup guide for the DOSBox Flatpak itself. :-)